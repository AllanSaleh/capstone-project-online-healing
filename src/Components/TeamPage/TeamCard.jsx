import React from 'react';

export default function TeamCard({ img, name, position }) {
  return (
    <div className="flex flex-col w-full my-4 lg:w-52 bg-blue-light rounded-3xl justify-between mx-2">
      <div className="lg:h-60">
        <img className="w-full" src={img} alt="MemberImage" />
        <div className="h-2 w-full bg-green-300" />
      </div>
      <div className="h-48 flex flex-col justify-around items-center">
        <div className="text-gray-700 text-3xl lg:text-subtitle text-center">{name}</div>
        <div className="text-gray-700 text-lg lg:text-paragraph text-center w-40 lg:w-auto">{position}</div>
      </div>
    </div>
  );
}
