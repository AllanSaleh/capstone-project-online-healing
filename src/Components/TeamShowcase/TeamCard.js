import React from 'react';

export default function TeamCard({ img, name, position }) {
  return (
    <div className="flex flex-col w-40 mt-2 lg:w-52 bg-blue-light rounded-3xl justify-between">
      <div className="lg:h-60">
        <img className="lg:w-full" src={img} alt="MemberImage" />
        <div className="h-2 w-full bg-green-300" />
      </div>
      <div className="h-44 flex flex-col justify-evenly">
        <div className="text-gray-700 text-lg lg:text-subtitle text-center">{name}</div>
        <div className="text-gray-700 text-lg lg:text-paragraph text-center">{position}</div>
      </div>
    </div>
  );
}
