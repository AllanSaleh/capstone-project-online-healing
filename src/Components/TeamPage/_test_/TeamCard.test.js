import * as React from 'react';
import TeamCard from "../TeamCard";
import * as renderer from "react-test-renderer";

it('Team Card renders correctly', () => {
  const tree = renderer
    .create(<TeamCard/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});