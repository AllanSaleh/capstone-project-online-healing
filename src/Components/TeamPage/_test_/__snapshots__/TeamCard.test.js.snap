// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex flex-col w-40 mt-2 lg:w-52 bg-blue-light rounded-3xl justify-between mx-2"
>
  <div
    className="lg:h-60"
  >
    <img
      alt="MemberImage"
      className="lg:w-full"
    />
    <div
      className="h-2 w-full bg-green-300"
    />
  </div>
  <div
    className="h-48 flex flex-col justify-around"
  >
    <div
      className="text-gray-700 text-lg lg:text-subtitle text-center"
    />
    <div
      className="text-gray-700 text-lg lg:text-paragraph text-center"
    />
  </div>
</div>
`;
